src/controllers/users.controller.js

```diff
@@ -68,7 +68,7 @@ const User = {
   },
 
   async updateProfile(req, res) {
-    const { email, role, ...updateData } = {
+    const { email, role, RoleId, createdAt, updatedAt, id, ...updateData } = {
```

Comment:
Thank you @lemoissonM. we can think also the data like `isVerified`, `rememberMe`,